index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="vewport" content="width=device-width, initial-scale=1.0">
+    <title>Documents</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+ <boby>
+    <nav class="navbar">
+        <div class="name">Patrick & Amos</div>
+        <div class="links">
+            <a href="#about">About</a>
+            <a href="#project">Project</a>
+            <a href="#contact">Contact</a>
+        </div>
+    </nav>
+
+    <div class="main-section">
+        <section id="about">
+            <header class="header">
+                <h1>About</h1>
+                <h2>Patrick & Amos</h2>
+                <h2>Web Developer</h2>
+            </header>
+            <div class="aboutme">
+                <p>
+                    we are skilled web developer that enjoys being practical in our projects
+                </p>     
+            </div>
+                <section id="projects">
+                    <header class="header">
+                        <h1>Projects</h1>
+                        <h2>Here are some of the projects we made</h2>
+                    </header>
+                    <div class="project">
+                        <div class="project-title">Official Logo</div>
+                        <img src="images/IMG-20210410-WA0001 (1).jpg" alt="survey" class="project-img">
+                    </div>
+                    <div class="project">
+                        <div class="project-title">Logo Design</div>
+                        <img src="images/IMG-20210410-WA0002.jpg" alt="survey" class="project-img">
```

Comment:
The images seem not to appear, I would suggest that you put the images in the folder( named images for simplicity). So when you want to add the image to your website the html can refer to the folder with the images such as `./images` 😄 
![200317082-5c8c4342-3595-4c2c-a821-a51c4e4eb1f7](https://user-images.githubusercontent.com/106984562/227925893-27d6a628-c306-4d79-b312-84d87b7a8968.jpg)